packages/camera/pubspec.yaml

```diff
@@ -16,6 +16,7 @@ homepage: https://github.com/flutter/plugins/tree/master/packages/camera
 dependencies:
   flutter:
     sdk: flutter
+  device_info: ^0.4.0+2
```

Comment:
I'm not sure if this is the best way to include this dependency. If someone uses a different version of `device_info` and `camera`, this might break. However, I think the `all_plugins_apk` test should prevent them from being incompatible thought.